vyper/evm/opcodes.py

```diff
@@ -171,8 +172,8 @@
     "INVALID": (0xFE, 0, 0, 0),
     "DEBUG": (0xA5, 1, 0, 0),
     "BREAKPOINT": (0xA6, 0, 0, 0),
-    "TLOAD": (0x5C, 1, 1, 100),
-    "TSTORE": (0x5D, 2, 0, 100),
+    "TLOAD": (0x5C, 1, 1, (None, None, None, None, None, 100)),
+    "TSTORE": (0x5D, 2, 0, (None, None, None, None, None, 100)),
```

Comment:
Add (please cross-check with the [EIP](https://eips.ethereum.org/EIPS/eip-4844#opcode-to-get-versioned-hashes)):
```py
"HASH_OPCODE_BYTE": (0x49, 1, 1, (None, None, None, None, None, 3))
```

Furthermore, a quick question: why are `DEBUG` and `BREAKPOINT` part of this list and not `PSEUDO_OPCODES` in case used? Because the proposals have never been made it into the EVM:
- https://github.com/ethereum/EIPs/issues/201

Reply:
DEBUG and BREAKPOINT are used for vyper-specific tooling, see https://github.com/vyperlang/titanoboa/blob/03ec051bb230301c6484fe4dbd7c2b33fd86cbea/boa/__init__.py#L43-L47. DEBUG is used for `vdb`, not sure that is still maintained though